LearnArena/wrappers/ActionWrappers/clip_action_wrapper.py

```diff
@@ -0,0 +1,82 @@
+"""
+This code is from TextArena:
```

Comment:
Can you import the package without pasting the source code of the original package?